moirae/models/ecm/__init__.py

```diff
@@ -116,10 +129,11 @@ def update_transient_state(new_input: ECMInput,
                                   i_rc=iRC_kp1,
                                   hyst=hyst_kp1)
 
-    @staticmethod
-    def calculate_terminal_voltage(new_input: ECMInput,
-                                   transient_state: ECMTransientVector,
-                                   asoh: ECMASOH) -> ECMMeasurement:
+    def calculate_terminal_voltage(
+            self,
+            inputs: ECMInput,
```

Comment:
Isn't `input` a protected (but over-writable) name?

Reply:
Yes, so we use "inputs" here, which is not.